### PR TITLE
Changed mentions of 4.9 to 'future version'

### PR DESCRIPTION
**Pull request by opayne1**
A couple mentions of 4.9 made it into the 4.8 release notes. This PR fixes it to say 'future version' to leave room for modifications in the release. 

- Applies to `enterprise-4.8` only
- [Preview link #1](https://deploy-preview-36262--osdocs.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-8-release-notes?utm_source=github&utm_campaign=bot_dp#ocp-4-8-metering)
- [Preview link #2](https://deploy-preview-36262--osdocs.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-8-release-notes?utm_source=github&utm_campaign=bot_dp#ocp-4-8-builds-lasttriggeredimageid-parameter)

### CODE REVIEW

**Comment by opayne1** [comment]
/assign @sferich888 @xltian @vikram-redhat 
for approvals

**Review by kalexand-rh — COMMENTED** [review]
LGTM

**Comment by vikram-redhat** [comment]
I sorta agree with @sferich888.  But while these may be instances of a firm date/release, we still shouldn't put a number on it. It would be better to change (in these instances) to "next release" instead of a "future release".

**Comment by chargio** [comment]
LGTM

File: release_notes/ocp-4-8-release-notes.adoc [hunk 1]
```diff
@@ -950,7 +950,7 @@ New enhancements are available on the *Monitoring* -> *Dashboards* page in the {
 
 [id="ocp-4-8-metering"]
 === Metering
-The Metering Operator is deprecated as of {product-title} 4.6, and is scheduled to be removed in {product-title} 4.9.
+The Metering Operator is deprecated as of {product-title} 4.6, and is scheduled to be removed in {product-title} next release.
```
Comment (wking) [inline release_notes/ocp-4-8-release-notes.adoc:953]: nit:

> ... removed in OpenShift Container Platform next release.

reads a bit strangely to me.  I'd have expected:

> ... removed in the next release of OpenShift Container Platform.

or some such.

Comment (bergerhoffer) [reply release_notes/ocp-4-8-release-notes.adoc:953]: +1 I'd suggest either the wording that @wking proposed, or "the next {product-title} release"

**Pull request merged**